src/idobasicmenuitem.c

```diff
@@ -359,6 +359,12 @@ ido_basic_menu_item_new_from_model (GMenuItem    * menu_item,
 
   item = ido_basic_menu_item_new ();
 
+    IdoBasicMenuItemPrivate *p = ido_basic_menu_item_get_instance_private(IDO_BASIC_MENU_ITEM(item));
+    gboolean bUseMarkup = FALSE;
```

Comment:
The variable style seems to be different from what developers have used in the rest of the code (lower_case_with_underscores). We don't want to mix styles, do we?

Reply:
> The variable style seems to be different from what developers have used in the rest of the code (lower_case_with_underscores). We don't want to mix styles, do we?

Converted in https://github.com/AyatanaIndicators/ayatana-ido/pull/17/commits/b7b61e345a19cacb937cf0fb14398545ae8b9349